Src/PerformanceCollector/Shared/Implementation/QuickPulse/IQuickPulseServiceClient.cs

```diff
@@ -2,7 +2,8 @@
 {
     using System;
     using System.Collections.Generic;
-    using System.Security.Cryptography.X509Certificates;
+
+    using Microsoft.ApplicationInsights.Extensibility.Filtering;
```

Comment:
The feature is to calculate configurable metrics for people to view in QuickPulse :)
Originally this was Filtering because we did (and still do?) plan to reuse the same codebase in "mainland" SDK to be able to collect configurable (calculated) metrics in the regular pipeline, not just QuickPulse. We though of the feature as of "filtering" telemetry items to leave only the ones we're interested in. Then as the idea evolved, we have introduced projections, aggregation types, etc., so right now filtering is only the first stage of "calculating" a metric. At the same time, for full documents (that you see on the QuickPulse blade towards the right side), it's still just filtering and nothing else. So I'm personally fine with leaving it in as "filtering", but you might have other suggestions.